### PR TITLE
Implement fileable escrow

### PR DESCRIPTION
**Pull request by sunbreak1211**

### CODE REVIEW

**Review by telome — APPROVED** [review]
LGTM

File: deploy/TokenGatewayInit.sol
Comment (oldchili) [inline deploy/TokenGatewayInit.sol:29]: `escrow()` can be removed now.
This is also relevant for `op-token-bridge`.

**Review by oldchili — COMMENTED** [review]
Minor comment on a redundant interface, apart from that looks good.

File: deploy/TokenGatewayInit.sol
Comment (sunbreak1211) [reply deploy/TokenGatewayInit.sol:29]: Fixed, waiting for approval and I'll merge.

Comment (sunbreak1211) [reply deploy/TokenGatewayInit.sol:29]: Fixed on `op-token-bridge` as well: https://github.com/makerdao/op-token-bridge/pull/1/commits/a01b8725f20896390e63a6f65b109c25f8fb823c

**Review by oldchili — APPROVED** [review]

**Review by telome — APPROVED** [review]

**Pull request merged**
